cmd/main.go

```diff
@@ -21,15 +21,18 @@ import (
 
 	"sigs.k8s.io/kubebuilder/cmd/version"
 	"sigs.k8s.io/kubebuilder/pkg/cli"
+	pluginv0 "sigs.k8s.io/kubebuilder/pkg/plugin/v0"
 	pluginv2 "sigs.k8s.io/kubebuilder/pkg/plugin/v2"
```

Comment:
IMO we should not have pluginv0 for the same reasons described in https://github.com/kubernetes-sigs/kubebuilder/pull/1537#discussion_r432801716

See :
- v1 -> Project layout v1
- v2 -> Project layout v2
- v3 -> Project layout v3 (where the plugins started to be in place )

Keep clear the concept adopted: In the case of the v1 and v2 PROJECT version, a corresponding Plugin version is implied and constant --v1 implies thego.kubebuilder.io/v1 Plugin, and similarly for v2. This is for legacy purposes -- no such implication is made with the v3 PROJECT version.

So, for me replace it for Vo will not make easier or more intuitive the understatement. 
Also, what is the problem that will be solved with?